app/assets/stylesheets/partials/_start_page.scss

```diff
@@ -0,0 +1,104 @@
+// Place all the styles related to the StartPage controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+$highlight: 0 1px 0 #fff;
+
+.navbar-nav {
+  margin-top: 15px !important;
+}
+
+.start-background {
+  width: 100%;
+  max-height: 500px;
+  background: image-url("img/bg.jpg") 50% 0%;
+
+  h1 {
+    display: inline-block;
+    margin: 240px 0 0 0;
+    padding: 5px;
+    background: rgba(235, 113, 37, .75);
+    color: #fff;
+  }
+
+  p {
+    margin: 5px 0 40px 0;
+    padding: 5px;
+    background: rgba(0, 0, 0, .25);
+    color: #eee;
+    font-size: 15px;
+  }
+
+  @media (max-width: 768px) {
+    h1 {
+      margin-top: 40px;
+    }
+
+    p {
+      background: rgba(0, 0, 0, .5);
+    }
+  }
+}
+.second-background {
+  width: 100%;
+  max-height: 500px;
+  background: url(img/slide5.jpg) 50% 0%;
+
+  h1 {
+    display: inline-block;
+    margin: 240px 0 0 0;
+    padding: 5px;
+    background: rgba(235, 113, 37, .75);
+    color: #fff;
+  }
+
+  p {
+    margin: 5px 0 40px 0;
```

Comment:
Properties should be ordered background, color, font-size, margin, padding<br>Shorthand form for property `margin` should be written more concisely as `5px 0 40px` instead of `5px 0 40px 0`